client/signup/steps/plans/index.jsx

```diff
@@ -38,6 +38,30 @@ import './style.scss';
 
 export class PlansStep extends Component {
 	componentDidMount() {
+		if ( document && document.createElement && document.body ) {
+			if ( window.location.search ) {
+				// save this so that we can enter debug mode in the widget
+				window.salesteam_initial_search_string = window.location.search;
+			}
+
+			const salesTeamStyles = document.createElement( 'link' );
+			salesTeamStyles.setAttribute(
+				'href',
+				'//s0.wp.com/wp-content/a8c-plugins/wpcom-salesteam/css/wpcom-salesteam.css?ver=1'
+			);
+			salesTeamStyles.setAttribute( 'rel', 'stylesheet' );
+			salesTeamStyles.setAttribute( 'type', 'text/css' );
+			salesTeamStyles.setAttribute( 'media', 'all' );
+			document.body.appendChild( salesTeamStyles );
+
+			const salesTeamScript = document.createElement( 'script' );
+			salesTeamScript.setAttribute(
+				'src',
+				'//s0.wp.com/wp-content/a8c-plugins/wpcom-salesteam/js/wpcom-salesteam.js?ver=20190221'
```

Comment:
I confirmed the style & script loads, but the browser console shows the following warning. Maybe we need to use `widgets.wp.com` on these instead of `s0.wp.com`?

```
[Report Only] Refused to load the script 'https://s0.wp.com/wp-content/a8c-plugins/wpcom-salesteam/js/wpcom-salesteam.js?ver=20190221' because it violates the following Content Security Policy directive: "script-src 'self' 'report-sample' 'unsafe-eval' stats.wp.com https://widgets.wp.com *.wordpress.com https://apis.google.com 'nonce-f44104718222bb264d058382c739c3042f9582a062b9a12bac44c9b8a32fbc3cea5374873fc3e65637e55ef67e215dcc' www.google-analytics.com 'sha256-3yiQswl88knA3EhjrG5tj5gmV6EUdLYFvn2dygc0xUQ=' 'sha256-ZKTuGaoyrLu2lwYpcyzib+xE4/2mCN8PKv31uXS3Eg4='". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
```

Reply:
I was testing on `calypso.live` when this occurred.